tensorboard/webapp/app_module.ts

```diff
@@ -29,13 +29,15 @@ import {ROOT_REDUCERS, loggerMetaReducerFactory} from './reducer_config';
 import {HeaderModule} from './header/header_module';
 import {ReloaderModule} from './reloader/reloader_module';
 import {MatIconModule} from './mat_icon_module';
+import {FeatureFlagModule} from './feature_flag/feature_flag_module';
```

Comment:
nit: Sort imports by paths

Reply:
Since it is not enforceable or autoformattable, I hope not to make this a rule.

Reply:
OK. I"ve been trying to do this manually myself. But I agree it's better to find a way to automate it than to always do it manually.